programmers/문자열-내림차순으로-배치하기.js

```diff
@@ -0,0 +1,19 @@
+function solution(s) {
+  let answer = "";
+  let arr = [];
+  let strArr = [];
```

Comment:
배열 자체가 바뀌는 것이 아니라면 const로 선언해주세요

배열의 이름도 arr보다는 이 배열이 하람님의 함수에서 어떤 역할을 하는지 보다 명확한 이름이 있어야 합니다.

```suggestion
  const arr = [];
  const strArr = [];
```